Providers/Profile.swift

```diff
@@ -144,6 +144,16 @@ open class BrowserProfile: Profile {
 
         // Set up our database handles.
         self.loginsDB = BrowserDB(filename: "logins.db", secretKey: BrowserProfile.loginsKey, schema: LoginsSchema(), files: files)
+        
+        // Set security policy on the database
+        do {
+            var databasePath = URL(fileURLWithPath: (try! files.getAndEnsureDirectory())).appendingPathComponent("logins.db")
```

Comment:
Do you think the database file alone should be excluded or maybe whole `profile.profile` folder?

Reply:
I think I will change it to the entire folder. Now that I think about it, a db will also have a wal and shm file and other files associated with it (metadata).. In that case it's just safer to exclude the whole folder.